Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.6.1-alpine
+COPY main.py crawler.py config.py /home/python-sitemap/
+RUN mkdir -p /home/python-sitemap/output/
```

Comment:
Its maybe overkill to create folder « just » for the output ? (And its make mandatory to change the default config file)